### DISCUSSION TITLE
Ladekurve GO E

### DISCUSSION THREAD

**Discussion by Trunks1982** [camueller/SmartApplianceEnabler#231] [Allgemein]
würde sagen heute war die Ladekurve nahezu perfekt.

![0DEA0608-8BDA-4C43-8A87-4A198099A4BD](https://user-images.githubusercontent.com/62701386/159069573-86df4069-3d81-483f-92c8-bdc59bdf9f5d.jpeg)

**Comment by camueller** [comment]
Kann man sagen. Vermutlich mit der Version 1.6.20?

**Reply by Trunks1982** [reply]
Ja ist die neuste Version.Hatte zwar Anfangs wieder Probleme mit dem Ladevorgang, hat komischerweise immer abgebrochen aber einmal den SHM neu gestartet und dann gings los👍
